src/MessagePack.UnityClient/Assets/Scripts/Editor/PackageExporter.cs

```diff
@@ -11,14 +11,21 @@ public static void Export()
     {
         // configure
         var root = "Scripts/MessagePack";
-        var exportPath = "./MessagePack.Unity.unitypackage"; // missing version:)
+        var exportPath = "../../bin/MessagePack.unitypackage";
```

Comment:
Regarding the "missing version" comment, I noticed that recently too that in 1.x you released messagepack unitypackage files with the version built into the filename. And it wasn't obvious anywhere else what the version would be, so this seemed liked a good idea.
I don't know how/where you did that, but I can help with getting the version number to you if you can help with the part about building a file with that name.